### ISSUE TITLE
Library Sub-String Search Problem

### ISSUE THREAD

**Issue by smeatheringham** [clementine-player/Clementine#6529]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [ x] I checked the issue tracker for similar issues
- [x ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/) - not done. I followed the instructions in the README.md file in the source code that pointed me to https://github.com/clementine-player/Clementine/wiki/Compiling-from-Source and that page does not seem to exist.

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Linux Mint 19.3
- Clementine version: 1.3.1

### Expected behaviour / actual behaviour
If I search my audio library for a complete word in summary:
Complete "words" are found.
Sub-strings that begin "words" are found.
Sub-strings within words or ending words are not found.

I had thought that I would look at the code and implement a fix. However, the code is completely undocumented and I do not feel comfortable reverse-engineering it.

### Steps to reproduce the problem (only for bugs)
If I search my audio library for a complete word, a string, say, "K456". Then it is found. 
If I am looking for a sub-string that occurs at the beginning of a word, say, "K45". Then it is found. It will pick up K456, K457, ...
However, if I am looking something that ends in "456" (because, for example, I cannot recall whether I have used K.456 or K456 in my naming) then it will not find anything ending in "456".

**Comment by jbroadus** [comment]
If you're still interested in digging into it, it looks like this is handled in the LibraryQuery class and LibraryQuery::Exec appears to submit the database query.

**Comment by smeatheringham** [comment]
Unfortunately, there are no instructions for compiling from source. Documentation (such as https://www.clementine-player.org/downloads) points to https://github.com/clementine-player/Clementine/wiki/Compiling-from-Source and that page does not seem to exist.

**Comment by hatstand** [comment]
Yeah, the link is broken but it's literally the first link on the wiki https://github.com/clementine-player/Clementine/wiki
